relatio/utils.py

```diff
@@ -56,7 +56,13 @@ def split_into_sentences(
         time.sleep(1)
         docs = tqdm(docs)
 
+    # Ready to be submitted now and test for merge
     for doc in docs:
+        if not (isinstance(doc["doc"], str)):
+            # Blank sentence (str format)
+            doc["doc"] = ""
+            doc_indices = doc_indices + [doc["id"]]
```

Comment:
Do we need this line? It also appears in line 68.

Reply:
When I wrote the test file for the change, it didn't determine the IDs for the blank strings for some reason, so I put this explicitly.

Reply:
It leads to problems. Here is a minimal example:
```
text = [{'id': 1, 'doc': 'This is a test. I have two sentences.'}, {'id': 2, 'doc': 'I have one sentence.'}, {'id': 3, 'doc': None}]
text = pd.DataFrame(text)
split_sentences = split_into_sentences(dataframe = text)
print(len(split_sentences[0]))
print(len(split_sentences[1]))
```

The number of ids does not match the number of split sentences, so we're not assigning sentences to the correct document id anymore. This is because `nlp("").sents` actually returns `None`.

I feel like the preferred option here would be to remove empty rows in the data frame and to add a warning message for the user: `"Your data frame had empty documents. Those were removed."`

Reply:
Right! Maybe I missed this. Anyway I had this conversation with Philine too, maybe the other alternative is what you said, write an explicit error message here so that it's easier for the user. Philine also suggested this.

Reply:
@Siddhant-Ray @PinchOfData 
A different approach can we define what we expect:
- id column has the type int
- doc column has the type str 

If this is not the case we throw a sensible error, and the user has to fix his input...
What do you think?

Reply:
Right, this can work well, as long as there is some feedback to the user.

Reply:
Do you see any downside? My view is that the users might feel uncomfortable with the input changed. So we can throw a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) and provide some actionable feedback.